packages/wrangler/src/__tests__/sentry.test.ts

```diff
@@ -131,12 +135,25 @@ describe("Error Reporting", () => {
 
     await reportError(new Error("test error"), "testFalse");
 
-    const { error_tracking_opt, error_tracking_opt_date } = TOML.parse(
-      await fsp.readFile(path.join(os.homedir(), reportingTOMLPath), "utf-8")
+    expect(
+      fs.existsSync(path.join(os.homedir(), reportingTOMLPath, "utf-8"))
+    ).toBe(false);
```

Comment:
Change to reporting does nothing if in a TTY or CI environment, test reflects that by confirming no Reporting TOML is created.